includes/header-functions.php

```diff
@@ -23,11 +25,16 @@ function get_header_images( $obj ) {
 		if ( $college ) {
 			$retval['header_image'] = get_field( 'degree_fallback_header_image', 'colleges_' . $college->term_id );
 			$retval['header_image_xs'] = get_field( 'degree_fallback_header_image_xs', 'colleges_' . $college->term_id );
+			$retval['header_image_md'] = get_field('degree_fallback_header_image_md', 'colleges_' . $college->term_id );			$retval['header_image_md'] = get_field('degree_fallback_header_image_md', 'colleges_' . $college->term_id );
```

Comment:
```suggestion
			$retval['header_image_md'] = get_field('degree_fallback_header_image_md', 'colleges_' . $college->term_id );
```

Looks like the same line got copied in twice.

Reply:
yes, that was a mistake I will remove it.